### ISSUE TITLE
Recent work has broken Windows (MinGW) build 

### ISSUE THREAD

**Issue by hoglet67** [stardot/b-em#5]
C:\MinGW\b-em>makebem.bat
windres.exe -i b-em.rc --input-format=rc -o b-em.res -O coff
C:\MinGW\bin\windres.exe: b-em.rc:145: syntax error
b-em.rc:4:0: fatal error: when writing output to : Invalid argument
compilation terminated.
C:\MinGW\bin\windres.exe: preprocessing failed.
make: *** [b-em.res] Error 1

**Comment by hoglet67** [comment]
I have fixed the about resource problem, and a couple of other things, see:
https://github.com/hoglet67/b-em/commit/0694b923cbc4ee947249b823764d8d37f80f16fb

I'm now stuck on some compilation issues in vdfs.c:

```
C:\MinGW\b-em>makebem.bat
make: `b-em.res' is up to date.
gcc.exe -O3 -DBEM  -DVERSION= -c scsi.c
gcc.exe -O3 -DBEM  -DVERSION= -c vdfs.c
vdfs.c: In function 'scan_entry':
vdfs.c:278:15: warning: incompatible implicit declaration of built-in function 'stpcpy' [enabled by default]
vdfs.c:281:11: warning: incompatible implicit declaration of built-in function 'stpcpy' [enabled by default]
vdfs.c:307:28: error: 'S_IRGRP' undeclared (first use in this function)
vdfs.c:307:28: note: each undeclared identifier is reported only once for each function it appears in
vdfs.c:307:36: error: 'S_IROTH' undeclared (first use in this function)
vdfs.c:309:28: error: 'S_IWGRP' undeclared (first use in this function)
vdfs.c:309:36: error: 'S_IWOTH' undeclared (first use in this function)
vdfs.c:311:28: error: 'S_IXGRP' undeclared (first use in this function)
vdfs.c:311:36: error: 'S_IXOTH' undeclared (first use in this function)
vdfs.c: In function 'write_back':
vdfs.c:523:15: warning: incompatible implicit declaration of built-in function 'stpcpy' [enabled by default]
vdfs.c:526:11: warning: incompatible implicit declaration of built-in function 'stpcpy' [enabled by default]
vdfs.c: In function 'open_file':
vdfs.c:544:15: warning: incompatible implicit declaration of built-in function 'stpcpy' [enabled by default]
make: *** [vdfs.o] Error 1
```
I'm guessing that S_IRGRP and similar symbols relate to some non-portable code.

Steve, any thoughts?

**Comment by hoglet67** [comment]
I've had a stab at fixing this, and I now have an executable that compiles and runs.

Steve, these changes need review:
https://github.com/hoglet67/b-em/commit/1c19d7755de3fd3c0ef4ad962d61974d6266825d

**Comment by hoglet67** [comment]
Thomas,  I've also had to hack the VERSION string by simply defining this as empty in the Makefile:
https://github.com/hoglet67/b-em/blob/db-test-windows/src/Makefile.win#L5

You can see the effect of this in the window title bar:
![image](https://cloud.githubusercontent.com/assets/1951463/23331136/a6eca4a0-fb57-11e6-9b83-9bac7567ab37.png)

What's the proper way to get this set up?

**Comment by ThomasAdam** [comment]
@hoglet67 -- the VERSION string is produced by running `utils/b-em-version-str.sh`.  Quite how you do this on Windows, I've no idea.

**Comment by ThomasAdam** [comment]
I see this now defaults to "Win32" --- whatever.

**Comment by hoglet67** [comment]
I'll open a new issue to cover improving the windows build process to give a correct version number.

**Comment by hoglet67** [comment]
#18
